first_year_scraper/excel_parser.py

```diff
@@ -0,0 +1,10 @@
+import camelot
+
+def parse_pdf(filename, target_filename):
+    print("Started processing pdf file. This might take a while...")
+    tables = camelot.read_pdf(filename, pages='1-end', copy_text=['h']) # if you are here to change args please be sure that you know what you are doing
+    tables.export(target_filename, f="excel")
+    print("Done parsing pdf. exported as: ", target_filename)
+
+if __name__ == "__main__":
+    parse_pdf("spr2019.pdf","test.xlsx")
```

Comment:
@iakshat I guess these calls are written to test the individual files? But I guess we don't need to have the aut2019.pdf and spr2019.pdf on the repository? @thealphadollar.

Reply:
I just kept them to make testing easy. We can remove it if we want after the review!😊

Reply:
I think it's cool to have it when someone tests locally! What do you say @sriyash421 ?

Reply:
Oh yes, we can keep them for local testing. @thealphadollar @iakshat